clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -728,6 +728,16 @@ runSYCLPostLinkTool(ArrayRef<StringRef> InputFiles, const ArgList &Args) {
   SmallVector<StringRef, 8> CmdArgs;
   CmdArgs.push_back(*SYCLPostLinkPath);
   const llvm::Triple Triple(Args.getLastArgValue(OPT_triple_EQ));
+  Arg *SYCLDeviceLibLoc = Args.getLastArg(OPT_sycl_device_library_location_EQ);
```

Comment:
We will need a test for this. Please look at existing tests to see how to run tests for clang-linker-wrapper.

Thanks

Reply:
Hi, @asudarsa 
The linker-wrapper tests have been added.
Thanks very much.